lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -435,6 +435,248 @@
+    width: 30%;
+    display: block;
+  }
+
+  label[for="email-students"]:hover + .email-students-hint {
+    display: block;
+  }
+
+  label[for="email-students-beta"]:hover + .email-students-beta-hint {
+    width: 30%;
+    display: block;
+  }
+
+  .enroll-actions {
+    margin-top: $baseline;
+  }
+
+  .member-lists-management {
+
+    .wrapper-member-select {
+      padding: ($baseline/2);
+      background-color: $light-gray;
+    }
+
+    .member-lists-selector {
+      display: block;
+      margin: ($baseline/4) 0;
+      padding: ($baseline/4);
+    }
+
+    .auth-list-container {
+      display: none;
+      margin-bottom: ($baseline*1.5);
+
+      &.active {
+        display: block;
+      }
+
+      .member-list-widget {
+
+        .header {
+          @include box-sizing(border-box);
+          @include border-top-radius(3);
+          position: relative;
+          padding: ($baseline/2);
+          background-color: #efefef;
+          border: 1px solid $light-gray;
+          display: none; // hiding to prefer dropdown as header
+        }
+
+        .title {
+          @include font-size(16);
+        }
+
+        .label,
+        .form-label {
+          @extend %t-copy-sub1;
+          color: $lighter-base-font-color;
+        }
+
+        .info {
+          @include box-sizing(border-box);
+          padding: ($baseline/2);
+          border: 1px solid $light-gray;
+          color: $lighter-base-font-color;
+          line-height: 1.3em;
+          font-size: .85em;
+        }
+
+        .member-list {
+          @include box-sizing(border-box);
+
+          table {
+            width: 100%;
+          }
+
+          thead {
+            background-color: $light-gray;
+          }
+
+          tr {
+            border-bottom: 1px solid $light-gray;
+          }
+
+          td {
+            @extend %t-copy-sub1;
+            vertical-align: middle;
+            padding: ($baseline/2) ($baseline/4);
+            @include border-left(1px solid $light-gray);
+            @include border-right(1px solid $light-gray);
+            word-wrap: break-word;
+          }
+        }
+
+        .bottom-bar {
+          @include box-sizing(border-box);
+          @include border-bottom-radius(3);
+          position: relative;
+          padding: ($baseline/2);
+          margin-top: -1px;
+          border: 1px solid $light-gray;
+          background-color: #efefef;
+          box-shadow: inset #bbb 0px 1px 1px 0px;
+        }
+
+        // .add-field
+
+        input[type="button"].add {
+          @include idashbutton($blue);
+          position: absolute;
+          @include right($baseline);
+        }
+      }
+
+      .revoke {
+        color: $lighter-base-font-color;
+        cursor: pointer;
+
+        &:hover, &:focus {
+          color: $alert-color;
+        }
+      }
+    }
+  }
+}
+
+
+// view - cohort management
+// --------------------
+.instructor-dashboard-wrapper-2 section.idash-section#cohort_management {
```

Comment:
I think this is extending a pattern set in an earlier PR, but three quick notes here for the future: 
1- in general we try to avoid element based styles (section), so this would be dropped.
2- similarly, avoiding ID based styling  (#cohort_management) 
3- From a naming convention wrapper-2 doesn't help me understand what is being done here ( wrapper-instructor-dashboard)  might be a better name here.